packages/core/src/browser/style/drag-and-drop.css

```diff
@@ -0,0 +1,14 @@
+#draggable {
```

Comment:
it should go to `filesystem/src/browser/style/file-tree.css`, since these styles only make sense in the context of the file tree.

Reply:
Move the drag-and-drop.cs to filesystem/..../file-tree.css